demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -660,7 +661,7 @@ def test_get_error_ignore_list(self, mocker):
 
         validate_manager = ValidateManager()
         ignore_errors_list = validate_manager.get_error_ignore_list("fake")
-        assert ignore_errors_list['file_name'] == ['BA101', 'SC101', 'BA106']
+        assert ignore_errors_list['file_name'] == ['BA101', 'SC101', 'IN117', 'BA106', 'IN100']
```

Comment:
why is this modified?

Reply:
pay attention that in the ```validator_manager``` until now we collected **only** all the errors which can be ignored (we simply ignored from un-ignorable errors).

I changed the functionality in the way that we would get all the errors from the .pack-ignore including those who can not be ignored.

This is a test which checks that an un-ignorable errors won't be added to the ```ignore_errors``` list. 
hence modified it to include error codes which can not be ignored.
The logic is being handled in the ```base_validator``` now.